pallets/capacity/src/lib.rs

```diff
@@ -195,6 +195,18 @@ pub mod pallet {
 	#[pallet::getter(fn get_current_epoch_used_capacity)]
 	pub type CurrentEpochUsedCapacity<T: Config> = StorageValue<_, BalanceOf<T>, ValueQuery>;
 
+	#[pallet::type_value]
+	/// EpochLength defaults to T::MaxEpochLength when not set
+	pub fn EpochLengthDefault<T: Config>() -> T::BlockNumber {
+		T::MaxEpochLength::get()
```

Comment:
Since the Max is used for validation purposes, and I imagine it would be a big number, should we change the default to something else?

Reply:
We certainly can, currently it is set to 100 blocks, which is not a particularly large number

Reply:
Are you envisioning something like a configurable pallet constant `DefaultEpochLength` also set to something like 100? That would decouple it from `MaxEpochLength` while allowing us to scale up the max if we need to without messing with the default.